torch/ao/quantization/pt2e/utils.py

```diff
@@ -292,7 +292,7 @@ def _get_node_name_to_scope(model: GraphModule) -> Dict[str, Tuple[str, type]]:
         node_name_to_scope[n.name] = current_scope
     return node_name_to_scope
 
-def get_aten_graph_module(
+def get_aten_graph_module_for_pattern(
```

Comment:
if we're breaking BC anyway should we put an underscore before it while we're at it?